ui/src/app/components/pdf-viewer.spec.tsx

```diff
@@ -0,0 +1,27 @@
+import {shallow} from 'enzyme';
+import * as React from 'react';
+import {Document, Page} from 'react-pdf';
+
+import {PdfViewer, Props} from 'app/components/pdf-viewer';
+
+let props = {
+  windowSize: {width: 1700, height: 0},
+  pdfPath: '/assets/documents/fake-document-path.pdf'
+};
+
+it('should load PDF pages', async() => {
+  const wrapper = shallow(<PdfViewer {...props} />).shallow();
+
+  // Initially we should have a document and no pages.
+  expect(wrapper.find(Document).length).toEqual(1);
+  expect(wrapper.find(Page).length).toEqual(0);
+
+  // Simulate the PDF document loading and calling the 'onLoadSuccess' prop to indicate we have
+  // 10 pages in the PDF.
+  const pdfDocument = wrapper.find(Document);
+  const onSuccess = pdfDocument.prop('onLoadSuccess') as (data: object) => {};
+  onSuccess({numPages: 10});
```

Comment:
thinking out loud: Is there a way to test actual content on the pages? This kinda feels like we're saying "there are 10 pages. How many pages are there? 10? good." Not sure how much is really getting covered by this test I guess, but it may be all you can really do here.

Reply:
ah, looking at the test for the parent component, it seems like it tests some of this functionality, and the rest is covered by E2E tests.